src/rest.js

```diff
@@ -57,11 +57,18 @@ function del(config, auth, className, objectId) {
     }
     return Promise.resolve({});
   }).then(() => {
+    if (!auth.isMaster) {
+      return auth.getUserRoles();
```

Comment:
@flovilmart do you mean the auth.resolveACL will ask the server to get the new state of the user ?  
What does not do getUserRoles by using a cache if already fetched.

Sorry but my English is not perfect :)